wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -1278,11 +1290,6 @@ extension ZMUserSession: SyncAgentDelegate {
     }
 
     func processLegacyEvents() {
-        managedObjectContext.performGroupedBlock { [weak self] in
```

Comment:
suggestion: avoid executing processLegacyEvents when syncV2 here

This is called from other places.